### ISSUE TITLE
about  code of data augmentation 

### ISSUE THREAD

**Issue by zhaowenlovehome** [clguo/SA-UNet#2]
I see you code , can you updata your code of data augmentation .

**Comment by clguo** [comment]
Thank you for your attention, I have used different methods for data augmentation, different codes, the code is messy, but I will upload all the codes at the end of this month, I will upload the augmented dataset later, you can use it directly.

**Comment by clguo** [comment]
I have uploaded the datasets just now , you can use it directly

**Comment by song-cc** [comment]
I use your pretrained model to infer DRIVE test image. But I got the result which have many nose points. Can you tell me the step of test. 
![image](https://user-images.githubusercontent.com/54126770/134362531-3c12c6fd-9e67-4853-8ac2-ed0557327752.png)

**Comment by liul-ing** [comment]
> I have uploaded the datasets just now , you can use it directly

hi,i want to konw,in the keras_dataAug.py, randomCrop function,why the label is cropped,while the label is not?could you please reply me?Thank you!!
